.github/release-drafter.yml

```diff
@@ -1,5 +1,13 @@
-# See https://github.com/release-drafter/release-drafter#configuration
+categories:
+  - title: "Breaking Changes"
+    labels:
+      - "BC-break"
+  - title: "Major Features"
+    labels:
+      - "MAJOR"
+  - title: "Documentation enhancements"
```

Comment:
does this help somehow?

It will be actually good if we can exclude `small-fix` labels (like for typos - documenting these PRs is unwanted) and maybe doc updates too